aws/resource_aws_mwaa_environment.go

```diff
@@ -0,0 +1,652 @@
+		},
+
+		Schema: map[string]*schema.Schema{
+			"airflow_configuration_options": {
+				Type:      schema.TypeMap,
+				Optional:  true,
+				Sensitive: true,
+				Elem:      &schema.Schema{Type: schema.TypeString},
+			},
+			"airflow_version": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+			},
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"dag_s3_path": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"environment_class": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "mw1.small",
+			},
+			"execution_role_arn": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"kms_key": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validateArn,
+			},
+			"last_updated": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"created_at": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"error": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"error_code": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"error_message": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+								},
+							},
+						},
+						"status": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"logging_configuration": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"dag_processing_logs": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							MaxItems: 1,
+							Elem:     mwaaEnvironmentModuleLoggingConfigurationSchema(false),
+						},
+						"scheduler_logs": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							MaxItems: 1,
+							Elem:     mwaaEnvironmentModuleLoggingConfigurationSchema(false),
+						},
+						"task_logs": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							MaxItems: 1,
+							Elem:     mwaaEnvironmentModuleLoggingConfigurationSchema(true),
+						},
+						"webserver_logs": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							MaxItems: 1,
+							Elem:     mwaaEnvironmentModuleLoggingConfigurationSchema(false),
+						},
+						"worker_logs": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							MaxItems: 1,
+							Elem:     mwaaEnvironmentModuleLoggingConfigurationSchema(false),
+						},
+					},
+				},
+			},
+			"max_workers": {
```

Comment:
Relatedly, it appears the API now has a `MinWorkers` configuration available as well. Certainly don't mind getting in the existing code then following up with that addition.

```
    // The minimum number of workers that you want to run in your environment. MWAA
    // scales the number of Apache Airflow workers and the Fargate containers that
    // run your tasks up to the number you specify in the MaxWorkers field. When
    // there are no more tasks running, and no more in the queue, MWAA disposes
    // of the extra containers leaving the worker count you specify in the MinWorkers
    // field.
    MinWorkers *int64 `min:"1" type:"integer"`
```